docs/DeveloperGuide.md

```diff
@@ -760,17 +760,36 @@ otherwise)
 
       Use case resumes at step 1.
     
-* 1c. User inputs a number smaller than 0 or larger than 1,000,000.
+* 1c. User inputs a quantity number smaller than 0 or larger than 1,000,000.
 
     * 1c1. StoreMando shows an error message.
 
       Use case resumes at step 1.
 
-**Use case: UC2 - Delete an item in a specific location**
+**Use case: UC2 - Delete an item from the entire inventory**
```

Comment:
Should this just be "Delete an item from the inventory"? Having entire gives the idea that we have a feature that allows partial deletion.

Reply:
I added "entire" because we do have a delete feature that deletes from a specific location, which is like a sub-list of the entire inventory.